repos.bzl

```diff
@@ -834,8 +834,8 @@ def go_repositories():
         build_file_generation = "on",
         build_file_proto_mode = "disable",
         importpath = "github.com/knative/build",
-        sum = "h1:3KvuoUpgXYkK3WKaSBVN/0FSwRaukf212t2bQkMSWjY=",
-        version = "v0.3.1-0.20190330033454-38ace00371c7",
+        sum = "h1:o/VYWA3HKyZlNqdU2hDE5LHpanBe8gazgPKL97XJ6bo=",
```

Comment:
@fejta Why is this getting bumped instead of removed?

Reply:
It can be removed now, it came back when i fetched files from master to fix the dependencies issue.

Reply:
Because that is what is in go.sum

Reply:
@fejta i removed knative/build from go.sum and repos.bzl but now the job fails (https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/test-infra/15022/pull-test-infra-bazel/1190347627145728003).
Should i re add them ?

Reply:
Yes, I would just run `hack/update-deps.sh` and let it do its thing.


Something (most likely parts of pipeline) are depending on knative and therefore knative/build. Something like `go mod graph | grep /build` will help you dig into this (looks like aws-k8s-tester depends on it, for example), but I wouldn't spend a lot of time on this